wpilibNewCommands/src/test/native/cpp/frc2/command/CommandDecoratorTest.cpp

```diff
@@ -89,6 +89,10 @@ TEST_F(CommandDecoratorTest, AndThen) {
   EXPECT_TRUE(finished);
 }
 
+#ifdef __GNUC__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
 TEST_F(CommandDecoratorTest, Perpetually) {
```

Comment:
Should be changed to test `Endlessly` instead, if there isn't a test for it already.